### PR TITLE
Rename 'Team Win' to 'TeamWin' in 'TWRP' topic

### PR DESCRIPTION
**Pull request by Minionguyjpro**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

So that people find the developer on GitHub, as it now cannot be clicked because of the name 'Team Win'. If it is 'TeamWin', it could be clicked and goes directly to their profile

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/twrp/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-created_by: Team Win
+created_by: TeamWin
 display_name: TWRP (software)
 short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
```
Comment (kenyonj) [inline topics/twrp/index.md:4]: Should we update here too?

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: topics/twrp/index.md (hunk 1 above)
Comment (Minionguyjpro) [reply topics/twrp/index.md:4]: What do you mean?

Comment (kenyonj) [reply topics/twrp/index.md:4]:
```suggestion
short_description: TeamWin Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
```

Comment (kenyonj) [reply topics/twrp/index.md:4]: Although I see on https://twrp.me/about/ that they refer to themselves as `Team Win`. So I'm not sure either of these changes are necessary.

Comment (Minionguyjpro) [reply topics/twrp/index.md:4]: Yes, okay. Is there no way to redirect the 'Team Win' to the 'TeamWin' github profile?

Comment (kenyonj) [reply topics/twrp/index.md:4]: Ok, if that’s the case you can do this:
```suggestion
short_description: Team Win Recovery Project (TWRP), pronounced "twerp", is an open-source software custom recovery image for Android-based devices.
github_url: https://github.com/TeamWin
```

File: topics/twrp/index.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-created_by: Team Win
+created_by: TeamWin
```
Comment (kenyonj) [inline topics/twrp/index.md:2]: We can revert this also
```suggestion
created_by: Team Win
```

**Comment by Minionguyjpro** [comment]
Wait. I need to check a few things, because I don't know what it exactly did now.

**Comment by Minionguyjpro** [comment]
Okay. The GitHub URL is now added and the developer name is set to 'Team Win'. Ready to be merged.

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Review by kenyonj — APPROVED** [review]

**Pull request merged**